### PR TITLE
19/aboutuspage we care section

### PR DESCRIPTION
**Pull request by init-commit**
# Description

This pull request is for issue #19 and aims at creating the About Us page(we care section). We have worked in this issue with [Jane Wandia](https://github.com/Jane-Wandia).

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

wecare-whole
wecare-responsive-1
wecare-responsive-2

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

@IanOdhiambo9  Awaiting your review.

### CODE REVIEW

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 1]
```diff
@@ -21,6 +21,9 @@ import { NextDirective } from './directives/next.directive';
 
 import { PrevDirective } from './directives/prev.directive';
 import { AboutUsRoutingModule } from './about-us.routing';
+import { AboutUsWeCareComponent } from './components/about-us-we-care/about-us-we-care.component';
+import { ElewaVerticalIconAndTextComponent } from 'libs/features/components/cards/src/lib/cards/elewa-vertical-icon-and-text/elewa-vertical-icon-and-text.component';
+import { CardsModule } from '@elewa-group/features/components/cards';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:26]: fix the order of imports, This should be at the domain level above all components/directive imports

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 2]
```diff
@@ -39,12 +43,10 @@ import { AboutUsHistoryCarouselComponent } from './components/about-us-history-c
...
     AboutUsPageComponent,
+    AboutUsWeCareComponent,
     AboutUsHistoryCarouselComponent,
     ElewaAboutUsLocationSectionComponent
   ],
-  exports: [
-    TeamMembersCarouselComponent,
-    ElewaAboutUsLocationSectionComponent
-  ],
+  exports: [TeamMembersCarouselComponent, ElewaAboutUsLocationSectionComponent, AboutUsWeCareComponent, ElewaVerticalIconAndTextComponent],
```
Comment (LemmyMwaura) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:50]: No need to export the component you created and the verticalIconandtext component does not belong to this module so it should not be exported.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Make the necessary updates

File: libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts [hunk 3]
```diff
@@ -21,6 +21,9 @@ import { NextDirective } from './directives/next.directive';
 
 import { PrevDirective } from './directives/prev.directive';
 import { AboutUsRoutingModule } from './about-us.routing';
+import { AboutUsWeCareComponent } from './components/about-us-we-care/about-us-we-care.component';
+import { ElewaVerticalIconAndTextComponent } from 'libs/features/components/cards/src/lib/cards/elewa-vertical-icon-and-text/elewa-vertical-icon-and-text.component';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts:25]: Import from the aliased path, In your case you need to import the module that exports this component.

File: libs/pages/elewa/about-us/src/lib/components/about-us-we-care/about-us-we-care.component.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+          </p>
+        </div>
+      </div>
+
+      <div class="card-section">
+        <div class="cards" *ngFor="let card of cards">
+          <img [src]="card.icon" />
+          <h2 class="title">{{ card.title }}</h2>
+          <p>{{ card.description }}</p>
+        </div>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/components/about-us-we-care/about-us-we-care.component.html:32]: This issue was dependent on #14, it should therefore reuse that card.

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by IanOdhiambo9** [comment]
Closing because no updates have been made here since changes were requested

**Pull request closed without merging**